rdmorganiser/domain/rdmo.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" ?>
```

Comment:
why was encoding removed?

Reply:
The rdmo team removed it, will add it again.